include/networkit/io/EdgeListWriter.hpp

```diff
@@ -8,11 +8,8 @@
 #ifndef EDGELISTWRITER_H_
 #define EDGELISTWRITER_H_
 
-#include <fstream>
-#include <iostream>
-#include <string>
-
 #include <networkit/io/GraphReader.hpp>
```

Comment:
Sorry for bringing this up again: I believe `#include <networkit/io/GraphReader.hpp>` is not required.
Also while `fstream` was not required in the header, the implemenation needs it. So please include it in the .cpp file.

Reply:
I removed `fstream` because it is already included in `GraphWriter.hpp`. However, `GraphWriter.hpp` does not need `fstream`, only its derived classes use it. Should I move `fstream` to the derived classes and remove it from `GraphWriter.hpp`?

Reply:
Ah, I missed that. Sorry. Never mind then. While I try to not rely on indirect includes, it's quite common in networkit, so I guess it's fine.

Reply:
I think that there is no valid reason for including `fstream` in `GraphWriter.hpp`, and I agree that we should not rely on indirect includes. I'll move `fstream` to the derived classes.